### PR TITLE
modulesync 1.5.0

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+.travis.yml:
+  secure: ""
```
Comment (dhollinger) [inline .sync.yml]: Need to have a travis secret generated for travis test

File: metadata.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "crayfishx/rsyslog",
+  "name": "puppet-rsyslog",
   "version": "1.1.0",
   "author": "crayfishx",
```
Comment (bastelfreak) [inline metadata.json]: please change that to Vox Pupuli

Comment (dhollinger) [reply metadata.json]: module name has to be `puppet-rsyslog` since puppet is our forge namespace, no?

Comment (dhollinger) [reply metadata.json]: Or do you mean author?

**Pull request merged**